amplify api add-graphql-datasource fails with error: Cannot find module 'cloudform'

**Describe the bug**

`amplify api add-graphql-datasource` fails for a new Amplify API. 

```
Cannot find module 'cloudform'
Require stack:
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graphql-relational-schema-transformer/lib/RelationalDBTemplateGenerator.js
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graphql-relational-schema-transformer/lib/index.js
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/commands/api/add-graphql-datasource.js
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/index.js
- /usr/local/lib/node_modules/@aws-amplify/cli/lib/execution-manager.js
- /usr/local/lib/node_modules/@aws-amplify/cli/lib/index.js
- /usr/local/lib/node_modules/@aws-amplify/cli/bin/amplify
/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graphql-relational-schema-transformer/lib/RelationalDBTemplateGenerator.js:1
Error: Cannot find module 'cloudform'
Require stack:
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graphql-relational-schema-transformer/lib/RelationalDBTemplateGenerator.js
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graphql-relational-schema-transformer/lib/index.js
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/commands/api/add-graphql-datasource.js
- /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-category-api/index.js
- /usr/local/lib/node_modules/@aws-amplify/cli/lib/execution-manager.js
- /usr/local/lib/node_modules/@aws-amplify/cli/lib/index.js
- /usr/local/lib/node_modules/@aws-amplify/cli/bin/amplify
    at Object.<anonymous> (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/graphql-relational-schema-transformer/lib/RelationalDBTemplateGenerator.js:6:21)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
```

**Amplify CLI Version**
4.13.1

**To Reproduce**
1. `amplify add api` to set up a new API
2. `amplify push` to provision the API in the cloud
3. `amplify api add-graphql-datasource` generates error

**Expected behavior**
To continue setting up a GraphQL datasource for a new Amplify API.

**Desktop**
OS: Ubuntu 18.04
Node Version: 12.14.1

--- a/packages/graphql-relational-schema-transformer/package.json
+++ b/packages/graphql-relational-schema-transformer/package.json
@@ -23,6 +23,7 @@
     "clean": "rm -rf ./lib"
   },
   "dependencies": {
+    "cloudform": "^4.1.0",
     "cloudform-types": "^4.1.0",
     "fs-extra": "^8.1.0",
     "graphql": "^14.5.8",
@@ -33,7 +34,6 @@
     "@types/fs-extra": "^8.0.1",
     "@types/jest": "23.1.1",
     "aws-sdk": "^2.608.0",
-    "cloudform": "^4.1.0",
     "jest": "^23.1.0",
     "ts-jest": "^22.4.6"
   },

--- a/packages/graphql-transformers-e2e-tests/package.json
+++ b/packages/graphql-transformers-e2e-tests/package.json
@@ -35,7 +35,7 @@
     "@types/jest": "^25.1.1",
     "@types/node": "^10.17.13",
     "aws-amplify": "^2.2.2",
-    "aws-appsync": "^3.0.2",
+    "aws-appsync": "^2.0.2",
     "aws-sdk": "^2.608.0",
     "graphql-auth-transformer": "6.13.1",
     "graphql-connection-transformer": "4.13.1",

